collectors/salesforce/cfn/collector.template

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
AFAICS you can remove this file. The base template shouldn't be required here

Reply:
+1